src/sortedIndexBy.ts

```diff
@@ -71,12 +70,6 @@ export function sortedIndexBy<T>(
  * duplicates. If the item already exists in the array the index would be of the
  * first occurrence of the item.
  *
- * @param data - The (ascending) sorted array.
- * @param item - The item to insert.
- * @param valueFunction - All comparisons would be performed on the result of
- * calling this function on each compared item. Preferably this function should
- * return a `number` or `string`. This function should be the same as the one
- * provided to sortBy to sort the array.
```

Comment:
This doesn't seem right, what happened here?

Reply:
ah, i think it's because this jsdoc isn't considered the jsdoc of the function, because of the newline... let me fix

Reply:
oh, i think this jsdoc shouldn't exist. i'll remove it.

Reply:
(there are jsdocs for the overloads, not this)